.github/actions/step_sdk_setup/action.yml

```diff
@@ -13,44 +13,42 @@ runs:
   - name: Clean up installed packages
     working-directory: ${{ inputs.scriptPath }}
     continue-on-error: true
-    shell: bash -el {0}
-    run: |-
-      conda activate release-env
+    shell: pwsh
+    run: |
       pip uninstall -y promptflow promptflow-sdk promptflow-tools
   - name: 'Build and install: promptflow-tools'
-    shell: bash -el {0}
-    run: |-
-      conda activate release-env
+    shell: pwsh
+    run: |
+      Set-PSDebug -Trace 2
```

Comment:
What does this command do?

Reply:
same with set -x.
It will print your command first and then print the output.
2 is the most verbose way. It will print all the execution flow for one line of powershell
![image](https://github.com/microsoft/promptflow/assets/2208599/bfb91339-fb00-4ef5-beb0-8f21362a4fc6)